src/App/Web/Polls/Controllers/PollVoteController.php

```
@@ -0,0 +1,84 @@
+use App\Web\Polls\Requests\StorePollVoteRequest;
+use Domain\Events\Models\Event;
+use Domain\Polls\Models\Poll;
+use Domain\Polls\Models\PollVote;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Collection;
+use Illuminate\Validation\ValidationException;
+use Support\Controllers\Controller;
+
+class PollVoteController extends Controller
+{
+    public function store(Event $event, Poll $poll, StorePollVoteRequest $request): RedirectResponse
```

PHPMD (2.15.0)

Avoid unused parameters such as '$event'. (undefined)

(UnusedFormalParameter)